### PR TITLE
feat: allow replies

### LINKED ISSUE #16: allow replies to comments
- Add a button to reply to a comment
- Link comments using a property like `parentCommentId`
- in the rendering filter comments to see if there are replies

### PR DESCRIPTION
**Pull request by spaenleh**
Add replies to comments

closes #16

### CODE REVIEW

File: src/components/common/CodeReview.js [hunk 1]
```diff
@@ -218,29 +250,47 @@ class CodeReview extends Component {
...
-            onCancel={this.handleCancel}
-            onSubmit={(_id, line, content) =>
-              this.handleSubmit(_id, line, content)
-            }
-          />
-        </td>
-      </tr>
+    const levelComments = comments
+      .filter((comment) => comment.data.parent === parentId)
+      .sort((comment) => comment.created_at);
```
Comment (juancarlosfarah) [inline src/components/common/CodeReview.js:258]: I'm pretty sure it's `createdAt`. You can check in the network tab.

File: src/components/common/CodeReview.js [hunk 2]
```diff
@@ -218,29 +250,47 @@ class CodeReview extends Component {
...
-            onEditComment={(_id) => this.handleEdit(_id)}
-            onDeleteComment={(_id) => this.handleDelete(_id)}
-            onCancel={this.handleCancel}
-            onSubmit={(_id, line, content) =>
-              this.handleSubmit(_id, line, content)
-            }
-          />
-        </td>
-      </tr>
+    const levelComments = comments
```
Comment (juancarlosfarah) [inline src/components/common/CodeReview.js:256]: Why call them `levelComment` and not `childrenComments`?

File: src/components/common/CodeReview.js [hunk 3]
```diff
@@ -218,29 +250,47 @@ class CodeReview extends Component {
     return comment.type !== COMMENT;
   }
 
-  renderCommentList(commentList) {
+  renderCommentChild(comments, parentId) {
```
Comment (juancarlosfarah) [inline src/components/common/CodeReview.js:253]: I would call this method `renderCommentChildren` or `renderChildrenComments`.

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looks good! Just some minor comments. Also, I would add in the renderer that if the comment has been deleted, you translate the content (so that we can see `[DELETED]` in either English or French).

File: src/components/common/CodeReview.js (hunk 1 above)
Comment (spaenleh) [reply src/components/common/CodeReview.js:258]: Yes you are right !

**Review by juancarlosfarah — APPROVED** [review]
🚀

**Pull request merged**